### ISSUE TITLE
Samsung S4Mini I9195 (serranoltexx)

### ISSUE THREAD

**Issue by BlauerKlausR** [Lanchon/REPIT#103]
device Samsung S4Mini I9195  
device codename serranoltexx
recovery  twrp_3.3.1-0
kernel 3.4.113-gbf6ef2d
rom lineage-16.0-20191212-UNOFFICIAL-serranoltexx.zip
[repit-dump--I9195.log](https://github.com/Lanchon/REPIT/files/4006916/repit-dump--I9195.log)
is your device running the stock partition layout or is it already modified?: no modification

**Comment by Lanchon** [comment]
is anybody still interested in this device?

**Comment by piero-e** [comment]
yes i am interrested :  https://github.com/Lanchon/REPIT/issues/118
